sweepbatcher/store.go

```diff
@@ -108,6 +111,12 @@ func (s *SQLStore) InsertSweepBatch(ctx context.Context, batch *dbBatch) (int32,
 	return s.baseDb.InsertBatch(ctx, batchToInsertArgs(*batch))
 }
 
+// DropBatch drops a batch from the database. Note that we only use this call
+// for batches that have no sweeps and so we'd not be able to resume.
+func (s *SQLStore) DropBatch(ctx context.Context, id int32) error {
```

Comment:
I propose to add a check that the number of sweep in the batch is 0. If the batch is not empty, return an error.

Reply:
Good idea, added!